nav2_follow_path_controller/src/nav2_follow_path_controller.cpp

```diff
@@ -12,66 +12,80 @@
 {
-  RCLCPP_INFO(get_logger(), "Creating");
+  RCLCPP_INFO(get_logger(), "Creating local planner");
 
   declare_parameter("controller_frequency", 20.0);
 
-  // The costmap node is used in the implementation of the DWB controller
-  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>(
-    "local_costmap", nav2_util::add_namespaces(std::string{get_namespace()}, "local_costmap"));
+  // The costmap node is used in the implementation of the local planner
+  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>("local_costmap");
+
+  // Create an executor that will be used to spin the costmap node
+  costmap_executor_ = std::make_unique<rclcpp::executors::SingleThreadedExecutor>();
 
   // Launch a thread to run the costmap node
   costmap_thread_ = std::make_unique<std::thread>(
     [&](rclcpp_lifecycle::LifecycleNode::SharedPtr node)
     {
       // TODO(mjeronimo): Once Brian pushes his change upstream to rlcpp executors, we'll
       // be able to provide our own executor to spin(), reducing this to a single line
-      costmap_executor_.add_node(node->get_node_base_interface());
-      costmap_executor_.spin();
-      costmap_executor_.remove_node(node->get_node_base_interface());
+      costmap_executor_->add_node(node->get_node_base_interface());
+      costmap_executor_->spin();
+      costmap_executor_->remove_node(node->get_node_base_interface());
     }, costmap_ros_);
 }
 
-DwbController::~DwbController()
+nav2_follow_path_controller::~nav2_follow_path_controller()
 {
   RCLCPP_INFO(get_logger(), "Destroying");
-  costmap_executor_.cancel();
+  costmap_executor_->cancel();
   costmap_thread_->join();
 }
 
 nav2_util::CallbackReturn
-DwbController::on_configure(const rclcpp_lifecycle::State & state)
+nav2_follow_path_controller::on_configure(const rclcpp_lifecycle::State & state)
 {
-  RCLCPP_INFO(get_logger(), "Configuring");
+  std::string local_planner_name;
+
+  RCLCPP_INFO(get_logger(), "Configuring local planner interface");
 
   costmap_ros_->on_configure(state);
 
   auto node = shared_from_this();
 
   progress_checker_ = std::make_unique<ProgressChecker>(rclcpp_node_);
 
-  planner_ = std::make_unique<dwb_core::DWBLocalPlanner>(
-    node, costmap_ros_->getTfBuffer(), costmap_ros_);
-  planner_->on_configure(state);
+  try {
+    // TODO(yathartha3): Use Parameter server correctly
+    local_planner_ = blp_loader_.createUniqueInstance("dwb_planner::DWBLocalPlanner");
+    RCLCPP_INFO(get_logger(), "Created local_planner");
+    local_planner_->initialize(node, costmap_ros_->getTfBuffer(), costmap_ros_);
+    local_planner_->on_configure(state);
+  } catch (const pluginlib::PluginlibException & ex) {
+    RCLCPP_INFO(get_logger(), "Failed to create local planner. Exception: %s", ex.what());
+    RCLCPP_INFO(
+      get_logger(),
+      "Make sure it is properly registered and that the containing library is built.");
```

Comment:
Please delete this line. I think you added this because you're used to seeing it when things fail but that line comes from the pluginlib exceptions so it'll be stated twice.